animareflection/ui
# Banner Component

## Description

##### Task link: https://trello.com/c/C9mrl9bu/103-banner-component

Added `Banner` component.

## Test Steps

Ensure the component's structure, stories, and testing suite are robust.
Confirm that all tests are successful.
Check that the Banner displays correctly in both storybook and sample downstream applications.
